From: David Cohen <david.a.cohen@intel.com>
Subject: [PATCH] usb: dwc3-pci: REMOVEME: set phy's platform device parent

Phy needs to request ACPI resources from OTG driver temporarily. Let's set
device's parent field as DWC3 PCI driver.

Signed-off-by: David Cohen <david.a.cohen@intel.com>
---
Index: linux/drivers/usb/dwc3/dwc3-pci.c
===================================================================
--- linux.orig/drivers/usb/dwc3/dwc3-pci.c	2014-06-23 17:30:43.194323052 -0700
+++ linux/drivers/usb/dwc3/dwc3-pci.c	2014-06-23 17:49:05.310048902 -0700
@@ -49,6 +49,7 @@
 	pdev = platform_device_alloc("usb_phy_gen_xceiv", 0);
 	if (!pdev)
 		return -ENOMEM;
+	pdev->dev.parent = glue->dev;
 
 	glue->usb2_phy = pdev;
 	pdata.type = USB_PHY_TYPE_USB2;
